### PR TITLE
V74 x dev

### PR DESCRIPTION
**Pull request by monttj**
1. Minor update for cleaning of input source to avoid any confusion as to which sources are being used for CAT object.
2. remove ak4PFJets sequence as it complains when running over AODSIM. Do we really need to rebuild the ak4PFJets?

### CODE REVIEW

File: CatProducer/python/catTools_cff.py [hunk 1]
```diff
@@ -23,8 +23,8 @@ def catTool(process, runOnMC=True, doSecVertex=True, useMiniAOD = True):
     process.load("CATTools.CatProducer.recoEventInfo_cfi")
 #######################################################################    
 # adding pfMVAMet
-    process.load("RecoJets.JetProducers.ak4PFJets_cfi")
-    process.ak4PFJets.src = cms.InputTag("packedPFCandidates")
+    #process.load("RecoJets.JetProducers.ak4PFJets_cfi")
```
Comment (jshlee) [inline CatProducer/python/catTools_cff.py]: this is the recipe for pfmvamet for miniaod

Comment (jhgoh) [inline CatProducer/python/catTools_cff.py]: This must be the circular dep. issue in AODSIM I mentioned this morning.

**Comment by jshlee** [comment]
shall we merge?

**Comment by monttj** [comment]
Yes, please merge it. Thank you!

**Pull request merged**